spec/models/user_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  user = User.new(Name: 'Tome', Bio: 'teacher', Photo: 'https://unsplash.com/photos/F_-0BxGuVvo')
+
+  it 'should be valid if it has a name' do
+    expect(user).to be_valid
+  end
+
+  it 'should not be valid if the name is blank' do
+    user.Name = ''
+    expect(user).to_not be_valid
+    user.Name = 'Tom'
```

Comment:
- [ ] Please write test statement under this declaration to be sure it's valid with correct values.